### PR TITLE
Explicitly hide tooltip when the window is deactivated (1-8-x)

### LINKED ISSUE #9943: Tooltip from minimized window impossible to dismiss
* Electron version: 1.7.1
* Operating system: Windows

### Expected behavior
Tooltip shouldn't be shown on the screen.

### Actual behavior
Minimize tooltip stays on the screen until electron app is brought into foreground

Same as - https://bugs.chromium.org/p/chromium/issues/detail?id=724538

### How to reproduce

Look into https://bugs.chromium.org/p/chromium/issues/detail?id=724538

### PR DESCRIPTION
**Pull request by brenca**
This is here to fix #9943

### CODE REVIEW

**Comment by alexeykuzmin** [comment]
@brenca If the master is affected, the bug should be fixes there too.
Can you please create a PR?

File: atom/browser/ui/win/atom_desktop_native_widget_aura.cc [hunk 1]
```diff
@@ -19,4 +23,21 @@ void AtomDesktopNativeWidgetAura::Activate() {
...
+    aura::Window* gained_active,
+    aura::Window* lost_active) {
+  views::DesktopNativeWidgetAura::OnWindowActivated(
+    reason, gained_active, lost_active);
+  if (lost_active != nullptr) {
+    auto* tooltip_controller = static_cast<views::corewm::TooltipController*>(
+      aura::client::GetTooltipClient(lost_active->GetRootWindow()));
+
+    // This will cause the tooltip to be hidden when a window is deactivated,
+    // as it should be. Remove this fix when the chromium issue is fixed.
```
Comment (alexeykuzmin) [inline atom/browser/ui/win/atom_desktop_native_widget_aura.cc]: > Remove this fix when the chromium issue is fixed.

It probably needs a `TODO` marker.

**Comment by brenca** [comment]
Sure, gonna create a PR there too ASAP.

**Comment by ckerr** [comment]
Merging.

@alexeykuzmin Since your X was procedural, ie to wait for the sibling PR to land in master, I'm going to merge this now that the sibling's also merged. If I've misread your X please feel free to reopen both here and in master.

**Pull request merged**